### PR TITLE
Address remaining review comments

### PR DESCRIPTION
**Pull request by wkozlowski-tudelft**

### CODE REVIEW

File: draft-irtf-qirg-principles-09-rc.xml [hunk 1]
```diff
@@ -301,16 +305,34 @@
...
      physics. We refer the reader to <xref target="SutorBook" /> and <xref
      target="NielsenChuang"/> for an in-depth introduction to quantum
-     information theory.</t>
+     information systems.</t>
+
+     <section title="Quantum state">
+
+       <t>A quantum mechanical system is described by its quantum state.
+       Mathematically speaking, a system's quantum state provides a probability
+       distribution for the outcomes of each possible measurement on that
```
Comment (brunorijsman) [inline draft-irtf-qirg-principles-09-rc.xml]: Maybe this is getting too deep into the weeds, but the quantum state is not _just_ a probability distribution. It also has a phase which causes the uniquely quantum phenomena interference. It's okay if you want to leave it as is.

File: draft-irtf-qirg-principles-09-rc.xml [hunk 2]
```diff
@@ -337,6 +359,14 @@
 
        <t>NOT (a |0> + b |1>) -> a |1> + b |0>.</t>
 
+       <t>It is important to note that "qubit" can have two meanings. In the
+       first meaning, "qubit" refers to a quantum *system* whose quantum state
+       can be expressed as a superposition of two basis states, which we often
+       label |0> and |1>. In the second meaning, "qubit" refers to the quantum
+       *state* of a quantum system with such two basis states. Both meanings
+       are used interchangeably in literature and the meaning is generally
+       clear from the context.</t>
```
Comment (brunorijsman) [inline draft-irtf-qirg-principles-09-rc.xml]: I can't quite follow what the distinction is.  The two meanings sound extremely similar to me.  Do you mean one meaning refers to the mathematical abstraction whereas the other meaning refers to the physical implementation?

File: draft-irtf-qirg-principles-09-rc.xml [hunk 3]
```diff
@@ -395,14 +425,19 @@
...
+       qubits provides a physically complete description of the two-qubit
+       system. The states of the two individual qubits are now correlated
+       beyond what is possible to achieve classically. Neither qubit is in a
+       definite |0> or |1> state, but if we perform a measurement on either
+       one, the outcome of the partner qubit will *always* yield the exact same
+       outcome. The final state, whether it's |00> or |11>, is fundamentally
+       random as before, but the states of the two qubits following a
+       measurement will always be identical. One can think of this as flipping
+       two coins, but the coins always both land on "heads" or both land on
+       "tails" together. Something that we know is impossible classically.</t>
```
Comment (brunorijsman) [inline draft-irtf-qirg-principles-09-rc.xml:455]: Nice!  This is the best lay-person description of entanglement that I have heard thus far!

File: draft-irtf-qirg-principles-09-rc.xml [hunk 4]
```diff
@@ -1548,22 +1580,37 @@ QR - quantum repeater
...
-           integrity of the transmitted qubits or the generated entanglement.
-           Instead, applications, such as QKD, establish such guarantees in an
-           end-to-end fashion using the classical network in conjunction with
-           the quantum one.
+           Fortunately, from an application's point of view, as long as the
+           underlying implementation corresponds to (or sufficiently
+           approximates) theoretical models of quantum cryptography, quantum
+           cryptographic protocols do not need the network to provide any
+           guarantees about the confidentiality or integrity of the transmitted
+           qubits or the generated entanglement. Instead, applications will
```
Comment (brunorijsman) [inline draft-irtf-qirg-principles-09-rc.xml]: RE> quantum cryptographic protocols do not need the network to provide any guarantees about the confidentiality or integrity of the transmitted qubits

But authentication is required (and this is non-trivial).  Specifically, the classical channel for post-processing is required to be an authenticated channel.

**Review by brunorijsman — APPROVED** [review]

**Comment by wkozlowski-tudelft** [comment]
Mailing list discussion: https://mailarchive.ietf.org/arch/msg/qirg/ibtdlv8b0gtDoGwME_TGaB3xAYc/
And resolved: https://mailarchive.ietf.org/arch/msg/qirg/HaLfN8sHXPIEutEGAS7H5I3qUzc/

**Pull request merged**